sysutils/nut/src/opnsense/service/templates/OPNsense/Nut/upsmon.conf

```diff
@@ -2,12 +2,12 @@
 # the next update.
 #
 {% if helpers.exists('OPNsense.Nut.usbhid.enable') and OPNsense.Nut.usbhid.enable == '1' %}
-MONITOR {{ OPNsense.Nut.usbhid.name }} 1 monuser {{ OPNsense.Nut.account.mon_password }} master
+MONITOR {{ OPNsense.Nut.general.name }} 1 monuser {{ OPNsense.Nut.account.mon_password }} master
 SHUTDOWNCMD "/sbin/shutdown -p +0"
 POWERDOWNFLAG /etc/killpower
 {% endif %}
 {% if helpers.exists('OPNsense.Nut.netclient.enable') and OPNsense.Nut.netclient.enable == '1' %}
-MONITOR {{ OPNsense.Nut.netclient.name }}@{{ OPNsense.Nut.netclient.address }} 1 {{ OPNsense.Nut.netclient.user }} {{ OPNsense.Nut.netclient.password }} slave
+MONITOR {{ OPNsense.Nut.general.name }}@{{ OPNsense.Nut.netclient.address }} 1 {{ OPNsense.Nut.netclient.user }} {{ OPNsense.Nut.netclient.password }} slave
 SHUTDOWNCMD "/sbin/shutdown -p +0"
```

Comment:
Maybe use the rc file here - also for the other one

Reply:
/usr/local/etc/rc.halt ?

Reply:
yes - the rc script is more stable because the services can stop themselves instead of getting killed. That increases the chance not to damage sqlite databases.

Reply:
Oh sqlite needs special attention, changed 👍